tensorflow/lite/micro/kernels/pack_test.cc

```diff
@@ -110,44 +110,11 @@ void TestPackThreeInputsFloat(int* input1_dims_data, const float* input1_data,
                       1e-5f, output_data);
 }
 
-void TestPackTwoInputsQuantized(
-    int* input1_dims_data, const int8_t* input1_data, int* input2_dims_data,
-    const int8_t* input2_data, int axis, int* output_dims_data,
-    const int8_t* expected_output_data, int8_t* output_data) {
-  TfLiteIntArray* input1_dims = IntArrayFromInts(input1_dims_data);
-  TfLiteIntArray* input2_dims = IntArrayFromInts(input2_dims_data);
-  TfLiteIntArray* output_dims = IntArrayFromInts(output_dims_data);
-  const int output_dims_count = ElementCount(*output_dims);
-
-  constexpr int input_size = 2;
-  constexpr int output_size = 1;
-  constexpr int tensors_size = input_size + output_size;
-  TfLiteTensor tensors[tensors_size] = {
-      // CreateQuantizedTensor needs scale/zero_point values as input, but these
-      // values don't matter as to the functionality of PACK, so just set as 1.0
-      // and 128.
-      CreateQuantizedTensor(input1_data, input1_dims, 1.0, 128),
-      CreateQuantizedTensor(input2_data, input2_dims, 1.0, 128),
-      CreateQuantizedTensor(output_data, output_dims, 1.0, 128)};
-
-  TfLitePackParams builtin_data = {
-      .values_count = 2,
-      .axis = axis,
-  };
-  int inputs_array_data[] = {2, 0, 1};
-  TfLiteIntArray* inputs_array = IntArrayFromInts(inputs_array_data);
-  int outputs_array_data[] = {1, 2};
-  TfLiteIntArray* outputs_array = IntArrayFromInts(outputs_array_data);
-
-  ValidatePackGoldens(tensors, tensors_size, builtin_data, inputs_array,
-                      outputs_array, expected_output_data, output_dims_count,
-                      1e-5f, output_data);
-}
-
-void TestPackTwoInputsQuantized32(
-    int* input1_dims_data, const int32_t* input1_data, int* input2_dims_data,
-    const int32_t* input2_data, int axis, int* output_dims_data,
-    const int32_t* expected_output_data, int32_t* output_data) {
+template <typename T>
+void TestPackTwoInputsQuantized(int* input1_dims_data, const T* input1_data,
```

Comment:
I understand that pack/unpack do not process any quantization information, and therefore this templating and code reorganization makes sense.  However, Google requires methods to do what they say, and this method no longer creates quantized tensors.  If you could please remove the `Quantized` from the method name?